test/language/expressions/async-arrow-function/early-errors-arrow-arguments-in-formal-parameters.js

```diff
@@ -0,0 +1,12 @@
+// Copyright 2016 Microsoft, Inc. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+author: Brian Terlson <brian.terlson@microsoft.com>
+es7id: pending
+description: It is a SyntaxError if FormalParameters contains arguments
+negative: SyntaxError
+---*/
+
+async(arguments) => {  }
```

Comment:
This isn't in the spec, is it meant to be? I just undid this check for general arrow functions